src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -146,7 +156,7 @@ public void AssignRowsAndInitializeDisplay()
                     counter.Row = row++;
                     if (counter.RenderValueInline)
                     {
-                        Console.WriteLine($"{name} {FormatValue(counter.LastValue)}");
+                        Console.WriteLine((_consoleWidth > CounterValueLength) ? $"{name} {FormatValue(counter.LastValue)}" : $"{FormatValue(counter.LastValue)}");
```

Comment:
A row should consist of the following:
Indent + name + " " + value

The length of the name is then changed depending on the width of the console. Thus, the value should always fit on the row. The following changes were made to make this clearer:
```c#
_maxNameLength = Math.Max(Math.Min(80, _consoleWidth) - (CounterValueLength + Indent + 1), 0);
string name = MakeFixedWidth($"{new string(' ', Indent)}{counter.DisplayName}", Indent + _maxNameLength);
```

Reply:
What happens when the console is 21 characters wide? When I simulate this in my head I got:
_maxNameLength = 0
name.Length = 4 (because it includes Indent)
Console.WriteLine("{name} {FormatValue(counter.LastValue}") is concatenating a 4 length thing, a 1 length space, and a value <= 20 length for a total possible length of 25. 25 chars wouldn't fit in the 21 character line.

I suspect the math will work out more cleanly and you can eliminate these corner cases if you define _maxNameLength to include every character on the line that isn't part of the 20 character value. Ie:

```C#
_maxNameLength = Math.Max(Math.Min(80, _consoleWidth) - CounterValueLength, 0);
string name = MakeFixedWidth($"{new string(' ', Indent)}{counter.DisplayName}", _maxNameLength);
Console.WriteLine("{name}{FormatValue(value)}");
```

Reply:
I initially included the indent and additional space between the name and the value into CounterValueLength, so having CounterValueLength = 20 was incorrect. I changed it to the correct value of CounterValueLength = 15. 

This means when the console is 21 characters wide, we should have the following:
_maxNameLength = 1
name.Length = 5
We would then be concatenating a 5-character string, a 1-length space, and 15-length value, resulting in a 21-length string.

Reply:
Ah I missed the change to CounterValueLength. OK so now it sounds like console widths >= CounterValueLength + Indent + 1 are guaranteed not to line wrap, and by changing CounterValueLength to 15 we've achieved the goal of supporting width >= 20. LGTM : )